src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbers {
+
+    private final static List<Number> lottoNumbers = createLottoNumbersCache();
```

Comment:
정적 상수 이므로 대문자로 되면 더 좋겠습니다 😄 
```suggestion
    private final static List<Number> LOTTO_NUMBERS = createLottoNumbersCache();
```

Reply:
감사합니다! 👍